### PR TITLE
Issue/2136 model-signature component

### PR DESCRIPTION
**Pull request by denyo**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://github.com/porscheofficial/oss-docs/blob/67c5b24838a293ce7a964884e6005eb71f2b8579/CONTRIBUTOR_LICENSE_AGREEMENT.md)

#### References

- Preview: https://designsystem.porsche.com/issue/2136/components/model-signature/examples

#### Scope

Short description of the pull request scope

### CODE REVIEW

**Comment by o-hook** [comment]
Hi @denyo ,
we really have to think about a solution how to expose the contents to screen readers as well. Both automatically and extendable through an `aria` property.

File: packages/components-js/public/index.html [hunk 1]
```diff
@@ -40,6 +40,7 @@
       <option value="link-social">Link Social</option>
       <option value="link-tile">Link Tile</option>
       <option value="marque">Marque</option>
+      <option value="model-signature">Model Signature</option>
```
Comment (denyo) [inline packages/components-js/public/index.html:43]: Order alphabetically after modal.

File: packages/components/src/components/model-signature/model-signature.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+};
+
+@Component({
+  tag: 'p-model-signature',
+  shadow: true,
+})
+export class ModelSignature {
+  @Element() public host!: HTMLElement;
+
+  /** Adapts the size of the component. */
```
Comment (denyo) [inline packages/components/src/components/model-signature/model-signature.tsx:27]:
```suggestion
  /** Adapts the model of the component. */
```

File: docker-compose.yml [hunk 3]
```diff
@@ -64,7 +64,7 @@ services:
       - utilities-deprecated-node-modules:/opt/porsche-design-system-app/packages/utilities-deprecated/node_modules
       - utilities-deprecated-utilities-node-modules:/opt/porsche-design-system-app/packages/utilities-deprecated/projects/utilities/node_modules
   change-volume-owner:
-    image: ghcr.io/porsche-design-system/porsche-design-system/playwright:v1.27.0-focal-v3-alpha-4
+    image: ghcr.io/porsche-design-system/porsche-design-system/playwright:v1.27.0-focal-v3-alpha-5
```
Comment (denyo) [inline docker-compose.yml:67]: Add volumes.

File: package.json [hunk 4]
```diff
@@ -63,7 +63,7 @@
...
-    "build:core-dependencies": "concurrently -n 'shared,fonts,fallbacks' 'yarn build:shared' 'yarn build:fonts' 'yarn build:fallbacks' && concurrently --kill-others-on-fail -n 'email,icons,marque,meta-icons,utilities,utilities-deprecated' 'yarn build:email' 'yarn build:icons' 'yarn build:marque' 'yarn build:meta-icons' 'yarn build:utilities' 'yarn build:utilities-deprecated' && concurrently -n 'assets' 'yarn build:assets'",
+    "build:core-dependencies": "concurrently -n 'shared,fonts,fallbacks,model-signatures' 'yarn build:shared' 'yarn build:fonts' 'yarn build:fallbacks' 'yarn build:model-signatures' && concurrently --kill-others-on-fail -n 'email,icons,marque,meta-icons,utilities,utilities-deprecated' 'yarn build:email' 'yarn build:icons' 'yarn build:marque' 'yarn build:meta-icons' 'yarn build:utilities' 'yarn build:utilities-deprecated' && concurrently -n 'assets' 'yarn build:assets'",
```
Comment (denyo) [inline package.json:66]: Move `email,icons,marque,meta-icons` into first concurrently.

File: packages/components-js/public/index.html [hunk 5]
```diff
@@ -12,8 +12,8 @@
...
+    <!--PLACEHOLDER_PORSCHE_DESIGN_SYSTEM_FONT_LATIN--><link rel=preload href=http://localhost:3001/fonts/porsche-next-w-la-regular.min.bdaf41e62559316d30f99df258039d4e.woff2 as=font type=font/woff2 crossorigin><link rel=preload href=http://localhost:3001/fonts/porsche-next-w-la-semi-bold.min.c24c2ecdded2c1342dba0f1d561dde72.woff2 as=font type=font/woff2 crossorigin><link rel=preload href=http://localhost:3001/fonts/porsche-next-w-la-bold.min.6b20b9525a53943efc0131a2def13eab.woff2 as=font type=font/woff2 crossorigin>
```
Comment (denyo) [inline packages/components-js/public/index.html:16]: Maybe rebuild utilities package.

File: packages/components-vue/index.html [hunk 6]
```diff
@@ -9,7 +9,7 @@
...
...-segmented-control,p-segmented-control-item,p-select-wrapper,p-spinner,p-stepper-horizontal,p-stepper-horizontal-item,p-switch,p-table,p-table-body,p-table-cell,p-table-head,p-table-head-cell,p-table-head-row,p-table-row,p-tabs,p-tabs-bar,p-tabs-item,p-tag,p-tag-dismissible,p-text,p-text-field-wrapper,p-text-list,p-text-list-item,p-textarea-wrapper,p-toast{visibility:hidden}.hydrated,.ssr{visibility:inherit}*{font-family:'Porsche Next','Arial Narrow',Arial,'Heiti SC',SimHei,sans-serif;line-height:calc(6px + 2.125ex)}p{font-weight:400}h1,h2,h3,h4,h5,h6{font-weight:400}a{color:inherit;text-decoration:underline}b,strong{font-weight:700}em,i{font-style:normal}a,button,input,select,textarea{outline:1px solid transparent;outline-offset:1px}a:focus,button:focus,input:focus,select:focus,textarea:focus{outline-color:currentColor}a:focus:not(:focus-visible),button:focus:not(:focus-visible),input:focus:not(:focus-visible),select:focus:not(:focus-visible),textarea:focus:not(:focus-visible){outline-color:transparent}@media(hover:hover){a{transition:color var(--p-transition-duration,.24s) ease}a:hover{color:rgba(148,149,152,0.20)}[data-theme="dark"] a:hover{color:rgba(126,127,130,0.20)}}</style>
```
Comment (denyo) [inline packages/components-vue/index.html:12]: Check font partial.

**Review by stephanschroeter — APPROVED** [review]

**Pull request merged**